docs/audits/zoom-audit.md

```diff
@@ -0,0 +1,68 @@
+# Zoom Audit
+
+## Background and Overview
+
+Zoom is one of the mechanisms used by vision impaired users to navigate through Web interfaces and programs. This is a builtin functionality in any Web browser that can offer from 25% up to 400% zoom by resizing the contents of a webpage. Given the importance of this tool in the accessibility context, WCAG has a complete set of guidelines regarding the expected behavior of webpages when zoom is used.
+
+The main premise of the WCAG standard is that websites need to preserve all the information regardless of the zoom level and the size of the user’s screen. The reality is that the way zoom is handled in webpages varies, and a lot of them will often break when more than 200% zoom is used.
+
+Given the importance of this tool for disabled users it was important to recognize the state of JupyterLab when used in higher zoom values. The following document will state the methodology, results and recommendations found in the audit.
+
+## Methodology
+
+Given that the JupyterLab interface is so complex, it was impossible to do one general audit of the whole interface. Instead, it was decided to break down the UI in the main 7 areas, left sidebar, right sidebar, main content area, menu bar, context menu, settings and the status bar.
+
+Each of the areas was tested in 0%, 200% and 400% zoom. In all the scenarios, JupyterLab default settings were used.
+
+## Findings
+
+All of the tests were made using JupyterLab 3.4.3 in a 16-inch Macbook Pro in Google Chrome.
```

Comment:
Do we really need to mention `a 16-inch Macbook Pro`? IMO language such as a `a laptop` or `desktop` would be more than enough

Reply:
This was copied directly from participants' notes. This is why I think copying them from there to here is iffy. 

Come to think of it, I may have left that note. I probably thought it was relevant because this was a zoom audit, and therefore screen size is potentially relevant.

Reply:
I left it because I felt that stating the screen size was important, the results may vary or other things may break if the screen size was bigger/smaller